clang/docs/OpenMPSupport.rst

```diff
@@ -580,6 +580,8 @@ implementation.
 | need_device_addr modifier for adjust_args clause            | :part:`partial`           | :none:`unclaimed`         | Parsing/Sema: https://github.com/llvm/llvm-project/pull/143442           |
 |                                                             |                           |                           |               https://github.com/llvm/llvm-project/pull/149586           |
 +-------------------------------------------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------+
+| need_device_ptr modifier for adjust_args clause             | :part:`partial`           | :none:`unclaimed`         | Clang Parsing/Sema: https://github.com/llvm/llvm-project/pull/168905     |
```

Comment:
It makes sense to have a row for `need_device_ptr` in the OpenMP 6.0 section, but there should also be one for the optional fallback argument in the OpenMP 6.1 section, and the PR link should go in that 6.1 row.